kDrive/UI/Controller/Menu/SharedWithMe/SharedDrivesViewController.swift

```diff
@@ -62,6 +62,12 @@ class SharedDrivesViewController: UIViewController {
             tableView.backgroundView = nil
         }
     }
+
+    static func instantiate() -> SharedDrivesViewController {
```

Comment:
I have a clearly overkill idea that's not a priority, but can we make this method generic (and therefore shared) for any ViewController specialized type ?

Reply:
We wanted to this but one issue we encountered is that we can't know the storyboard in advance